portfolio/src/main/webapp/life-map.js

```diff
@@ -0,0 +1,93 @@
+  let birthContent = '<div id="content">'+
+      '<h1 id="firstHeading">Step 1: The Hospital Where I was Born</h1>'+
+      '<div id="bodyContent">'+
+      '<p>On April 19, 2000, I was born at Littleton Adventist Hospital ' +
+      'with these character traits </p>' + 
+      '<ul>' +
+      '<li>Gender: Boy</li>' +
+      '<li>Weight: 6 pounds 3.4 ounces </li>' +
+      '<li>Height: 19 inches </li>' +
+      '<li>Time: 9:30PM </li>' +
+      '<li>Star Sign: Taurus (but Aires cusp) </li>' +
+      '<li>Rising Sign: Scorpio </li>' +
+      '<li>Chinese Year: Golden Dragon [heck yeah]</li>' +
+      '<li>Dexterity: 0</li>' +
+      '</div>'+
+      '</div>';
+
+  let childhoodHomeContent = '<div id="childhoodHomeContent">' +
```

Comment:
Formatting: Make sure to use consistent indentation in this file =)  Looks like this one and birthContent above are more indented than the others.

Reply:
redeployed

Reply:
Ran formatter